user.go

```diff
@@ -570,10 +572,20 @@ func (user *User) unlockedConnectWithCookies(cookies cookies.Cookies) (*messagix
 	}
 	user.log.Debug().Msg("Connecting to Meta")
 	// TODO set proxy for media client?
-	cli, err := messagix.NewClient(MessagixPlatform, cookies, log, proxyAddr)
-	if err != nil {
-		return nil, fmt.Errorf("failed to prepare client: %w", err)
+	attempts := 1
+	var cli *messagix.Client
+	for {
+		cli, err = messagix.NewClient(MessagixPlatform, cookies, log, proxyAddr)
+		if err != nil {
+			if attempts > maxConnectAttempts {
+				return nil, fmt.Errorf("failed to prepare client: %w", err)
+			}
```

Comment:
Oh right, this should also check that it's not a auth error (the ones listed in `Connect()`: token invalidated, challenge/consent required, LS error responses).

Or maybe check that it's a connection error specifically? That might be harder though